### PR TITLE
feat(model): add Message model

### PR DESCRIPTION
**Pull request by ImMPrada**
This PR adds the ability to get all messages of a `workspace_channel` and create more messages in a `workspace_channel`

image

### CODE REVIEW

File: app/controllers/messages_controller.rb
Comment (mapra99) [inline app/controllers/messages_controller.rb:1]: it looks very clean!

File: app/helpers/dates_helper.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module DatesHelper
+  def friendly_date(date)
+    today = Time.zone.today
+    date = date.to_date
+
+    return "today at #{date.strftime('%l:%M %p')}" if date == today
+
+    return "yesterday at #{date.strftime('%l:%M %p')}" if date == today - 1
```
Comment (mapra99) [inline app/helpers/dates_helper.rb]:
```suggestion
    return "today at #{date.strftime('%l:%M %p')}" if date == today
    return "yesterday at #{date.strftime('%l:%M %p')}" if date == today - 1
```

File: app/views/layouts/workspace_channel.html.erb [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;500;600;700&display=swap" rel="stylesheet">
```
Comment (mapra99) [inline app/views/layouts/workspace_channel.html.erb:9]: is this needed? I thought you were importing the fonts from the CSS file you had created

**Review by mapra99 — APPROVED** [review]

File: app/views/layouts/workspace_channel.html.erb (hunk 2 above)
Comment (ImMPrada) [reply app/views/layouts/workspace_channel.html.erb:9]: no no I deleted it from the CSS, the target is to remove that CSS file

File: app/helpers/dates_helper.rb (hunk 1 above)
Comment (ImMPrada) [reply app/helpers/dates_helper.rb]: done!

**Pull request merged**